<a name = 'top'></a>
# Collaborations
The Emerson Lab is a very collaborative group and has ongoing projects with many labs at UCI. Below are active collaborations we are currently pursuing on campus. Please stay tuned for external collaborations page.

* [*Drosophila* population genomics](#drosophilapopgen)
* [The evolution and genomics of butterfly vision](#butterflygenomics)
* [Genome assembly](#genomeassembly)
* [Fish physiology and genomics](#fishphysgen)
* [Experimental evolution](#experimentalevolution)

<a name = 'drosophilapopgen'></a>
## *Drosophila* population genomics

### Kevin Thornton

__We have constructed several platinum quality *Drosophila simulans* genome assemblies in order to study population variation of structural mutations and how these mutations affect gene expression.__

The [Thornton lab](http://www.molpopgen.org/) and the Emerson lab have a long standing interest in the patterns of variation associated with structural mutations, ie mutations that add, subtract, rearrange, or otherwise sculpt genome sequences. Such mutations can include tandem duplications, transposable element insertations, inversions, translocations, etc. Since we now routinely assemble platinum quality genomes from long molecule data, we can study SV in exquisite detail. [Building off of our recent work](https://academic.oup.com/nar/article/44/19/e147/2468393/Contiguous-and-accurate-de-novo-assembly-of), we have assembled four *D. simulans* genomes to a level of completeness and contiguity comparable to that exhibited by the [FlyBase reference genome](http://flybase.org/). The Thornton lab has generated expression data for these strains as well, and we are now characterizing the mutations. We have previously noted that [short read methods miss a significant proportion of structural variation](http://www.biorxiv.org/content/early/2017/03/08/114967). As a result, we are finding many mutations previously hidden from discovery. Our ultimate goal is to understand how SV segregating in natural populations affects the genome, including how it changes gene structure and transcription.

[[top]](#top)

### Tony Long
__We have assembled fourteen platinum quality genomes of _Drosophila melanogaster_ and are characterizing their structural mutations.__

The [Long lab](http://wfitch.bio.uci.edu/~tdlong/sandvox/) is interested in uncovering the genetic basis of natural variation in and dissecting the genetic architecture of complex traits, and has generated a mapping resource for this called the [*Drosophila* Synthetic Population Resource (DSPR)](https://flyrils.org) in collaboration with [Stuart Macdonald's lab](https://molecularbiosciences.ku.edu/stuart-j-macdonald). Together, the Long and Emerson labs have assembled fourteen genomes of *Drosophila melanogaster*, incuding the founders of the DSPR and Oregon-R. The DSPR also includes the widely used strains Canton-S and Samarkand. We are currently characterizing their genetic variation and studying the patterns of population variation in them in a project led by [Mahul Chakraborty](Chakraborty.md).

[[top]](#top)


<a name = 'butterflygenomics'></a>
## The evolution and genomics of butterfly vision

### Adriana Briscoe
__The [Briscoe lab](http://visiongene.bio.uci.edu/Adriana_Briscoe/Briscoe_Lab.html) works on the evolution, physiology, genetics, and genomics of butterfly color vision. The Emerson lab is currently working with the Briscoe lab to assemble the *Heliconius* genomes as part of an NSF funded project. We are also working on characterizing the functional genomics of *Heliconius* eyes, which have sexually dimorphic opsin expression.__

[[top]](#top)

<a name = 'genomeassembly'></a>
## Genome assembly

### Tony Long

__We have developed an experimental and computational pipeline for assembling genomes that fuses long molecule only assemblies with assemblies constructed from both short and long molecule data. This approach yields improvements in contiguity over both methods alone with minimal compromises to accuracy and completeness.__

The [Long lab](http://wfitch.bio.uci.edu/~tdlong/sandvox/) and the Emerson Lab have been avid adopters of long-molecule sequencing approaches (specifically Pacific Biosciences RS II P6/C4 chemistry) to assemble whole genomes. However, methods for assembling such data are still developing rapidly. Some approaches (like [Canu](https://github.com/marbl/canu)) employ only long molecules while others (like [DBG2OLC](https://github.com/yechengxi/DBG2OLC)) combine both long and short read data in methods know as "hybrid assembly". While for any particular data set, one method may perform better than another, they often appear to produce complementary assemblies. Mahul Chakraborty (Emerson lab) and Jim Baldwin-Brown (Long lab) have developed and evaluated a genome assembly approach called *[quickmerge](https://github.com/mahulchak/quickmerge)* for genome assembly that combines two haploid assemblies in an attempt to get the best of both worlds. In the process, we established a set of best practices for attaining accurate and contiguous metazoan assemblies, spanning DNA isolation, quality control, assembly, and polishing the assembly. Our work was published in 2016 in [*Nucleic Acids Research*](https://academic.oup.com/nar/article/44/19/e147/2468393/Contiguous-and-accurate-de-novo-assembly-of).

[[top]](#top)

<a name = 'fishphysgen'></a>
## Fish physiology and genomics

### Donovan German

__We are studying the molecular evolution and functional genomics of digestive physiology in the herbivorous monkeyface prickleback.__

The [German lab](http://german.bio.uci.edu/) specializes in comparative physiology of digestion, especially of fishes like the monkeyface prickleback, *Cebidichthys violaceus*. [Joe Heras](http://josephheras.weebly.com/) in the German lab is currently executing an ambitious project to understand the molecular basis of a recent transition from carnivory to herbivory observed in the prickleback. The goals include assembling a high quality reference genome, assembling a transcriptome, and annotating the genome. The Emerson lab is playing a supporting role in the project by advising and assisting in the reference genome assembly of *C. violaceus*. As the genomics resources are completed, we will study the molecular evolution of the digestive enzyme gene families.

[[top]](#top)

<a name = 'experimentalevolution'></a>
## Experimental evolution

### Brandon Gaut

__We are investigating the fitness effects of mutations and how understanding the distribution of such effects can help us build phenotypic models of adaptation. This work is being done in a [highly replicated experimental evolution experiment](http://science.sciencemag.org/content/335/6067/457) where *E. coli* was forced to adapt to high temperatures for 2,000 generations.__

[[top]](#top)

<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
