<!doctype html>
<html lang="en-US">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">

<!-- Begin Jekyll SEO tag v2.2.3 -->
<title>J.J. Emerson | [“Emerson Lab at UCI”]</title>
<meta property="og:title" content="J.J. Emerson" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="[“Evolutionary genetics lab at the University of California, Irvine”]" />
<meta property="og:description" content="[“Evolutionary genetics lab at the University of California, Irvine”]" />
<link rel="canonical" href="http://localhost:4000/people/Emerson.html" />
<meta property="og:url" content="http://localhost:4000/people/Emerson.html" />
<meta property="og:site_name" content="[“Emerson Lab at UCI”]" />
<script type="application/ld+json">
{"@context":"http://schema.org","@type":"WebPage","headline":"J.J. Emerson","description":"[“Evolutionary genetics lab at the University of California, Irvine”]","url":"http://localhost:4000/people/Emerson.html"}</script>
<!-- End Jekyll SEO tag -->

    <link href='https://fonts.googleapis.com/css?family=Arvo:400,700,400italic' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" href="/assets/css/style.css?v=6e966eaee6f8b81827fe6a0e5b4268e1c1f4a161">
    <script src="/assets/js/scale.fix.js"></script>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
  </head>
  <body>
    <div class="wrapper">
      <header>
        <h1 class="header">Emerson Lab at UCI</h1>
        <p class="header">Evolutionary genetics lab at the University of California&#44; Irvine</p>

        <ul>
          
	  <li><a href='/'>Home</a></li>
	  <li><a href='/research'>Research</a></li>
	  <li><a href='/people'>People</a></li>
	  <li><a href='/publications'>Publications</a></li>
	  <li><a href='/jobs'>Open Positions</a></li>
	  <li><a href='/collaborators'>Collaborators</a></li>
	  <li><a href='/samplingprocess'>Blog (alpha)</a></li>
	  <li><a class="buttons github" href="http://github.com/EmersonLab/emersonlab.github.io">View On GitHub</a></li>
        </ul>

        
          <p class="header">This project is maintained by <a class="header name" href="http://github.com/EmersonLab">EmersonLab</a></p>
        

        
      </header>

      <section>
        <h1 id="jj-emerson">J.J. Emerson</h1>

<h2 id="contact">Contact</h2>
<p>Twitter: <a href="https://twitter.com/JJ_Emerson">@JJ_Emerson</a><br />
<a href="/documents/J.J.EmersonCV.pdf">My CV</a><br />
<a href="http://www.faculty.uci.edu/profile.cfm?faculty_id=5987">UCI Contact Information</a></p>

<h2 id="academic-history">Academic History</h2>
<ul>
  <li>2013-present, Assistant Professor, Department of Ecology &amp; Evolutionary Biology, Center for Complexy Systems Biology, UC Irvine</li>
  <li>2010-2013, Postdoctoral Researcher with Doris Bachtrog, Integrative Biology, UC Berkeley</li>
  <li>2007-2010, Postdoctoral Fellow with Wen-Hsiung Li, Genomics Research Center, Academia Sinica, Taiwan</li>
  <li>2006, PhD Ecology &amp; Evolutionary Biology, University of Chicago</li>
  <li>2000, BA Ecology &amp; Evolution and Biochemistry, Rice University</li>
</ul>

<h2 id="publications">Publications</h2>
<p>Please see the Emerson lab <a href="/publications">publications page</a>.</p>

<h2 id="research-interests-and-approaches">Research Interests and Approaches</h2>
<p>I am interested in natural genetic variation, genomics, and evolution. In particular, I am interested in the evolution of new genes, the patterns and consequences of mutations that change genome structure, sex chromosome evolution, and the genetic architecture of gene expression variation. The projects that originate in my lab typically employ model organisms (e.g. fruit flies and baker’s yeast), though I frequently work on non-model systems through collaboration. I use the tools of population genetics and molecular evolution in order to understand the origin and fate of genetic variation within and between species.</p>

<h2 id="scientific-contributions">Scientific contributions</h2>

<p>Below are selected contributions I’ve made in the course of my career. See also the <a href="/publications">publications</a> page.</p>

<h3 id="evolution-of-structural-variation-in-drosophila">Evolution of structural variation in <em>Drosophila</em></h3>

<p>How does genome complexity arise? We know that complex modern genomes are descendants of much simpler ancestors and that throughout evolutionary history, mutation has added, subtracted, rearranged, or otherwise sculpted genome sequences. We also know this acquisition and loss of segments of genetic material (i.e. structural mutations) has proven particularly important for increasing genomic complexity, especially when new genes and new gene structures are produced. Our work in <em>Drosophila</em> contributed to the field of structural variation by producing the first analysis of copy number variation in populations at 100bp resolution in any organism. This previously unheard of precision was made possible by our marriage of high density microarrays with machine learning approaches. Our work established copy number as ubiquitous, even in species with compact genomes like Drosophila. It was also the first evolutionary analysis of copy number to rigorously estimate the influence of natural selection and genetic drift on such variants. We also identified duplication hotspots and discovered that these hotspots were associated with late replication. Finally, we also established a method to identify new genes in species without reference genomes.</p>

<h3 id="origin-of-new-genes">Origin of new genes</h3>

<p>The origin of new genes through duplication is a central mechanism by which the genetic repertoire of a genome expands. However, the forces that foster the preservation and differentiation of new genes was largely a mystery until full genome sequencing permitted the systematic study of new genes. By studying the origin of new genes through duplication via retrotransposition of mRNA in various vertebrates, we discovered a novel mode of new gene acquisition: biased gene traffic related to sex chromosomes. In our initial studies, we established the prevalence of gene copy export from the X- chromosome in mammals. In this research, we discovered that exported genes gained testis function far above the rate expected by chance and identified avoidance of germline X-inactivation for male related functions as a likely driver. We applied these same insights to the study of the chicken genome as well, discovering that differences in retrotransposon repertoire in the chicken genome likely limit the amount of gene traffic possible in that genome. As high throughput genome technology continued to mature, we began apply these newer method to the problem of new gene discovery. In particular, using comparative genomic hybridization with microarrays, we extended new gene discovery to species lacking reference genomes, as exemplified in new genes discovered in species closely related to <em>Drosophila melanogaster</em>.</p>

<h3 id="genetic-architecture-of-gene-expression-variation">Genetic architecture of gene expression variation</h3>

<p>Deciphering the relationship between genotype, molecular phenotype, and organismal phenotype is central to understanding evolution and natural selection. In particular, understanding the patterns and sources of molecular phenotypes like regulatory variation is crucial for understanding phenotypic and genome evolution. To study this, we developed deep sequencing methods using RNA-seq and measurement of allele specific expression (ASE) in F1 crosses to estimate gene expression variation between different yeast strains. Our work was the first to study cis and trans variation using ASE on a genome-wide scale using RNA-seq. This and subsequent work enabled us to characterize the rate, prevalence, and genetic architecture of expression variation. We also characterized natural selection and genetic drift acting on expression variation. By applying our methods to data from two very closely related strains of <em>Saccharomyces cerevisiae</em>, we classified the nature of expression variation architecture as either linked to a gene of interest (cis) or as a result of diffusible elements (trans). We found that expression variation is ubiquitous, with trans variation being more common within species and cis being more common between species. We also studied the dominance patterns of expression variation and discovered that trans variants are more likely to be dominant, whereas cis variants are more likely to exhibit co-dominant inheritance. Additionally, genes with antagonistic cis-trans interactions are more likely to exhibit over- or underdominant inheritance of expression level. Finally, we found that trans variation is more strongly constrained than cis variation and that cis variation is more likely to participate in adaptation.</p>


      </section>

      <footer>
        <!--p><small>Hosted on <a href="https://pages.github.com">GitHub Pages</a> using the Dinky theme</small></p-->
      </footer>
    </div>
    <!--[if !IE]><script>fixScale(document);</script><![endif]-->
    
  </body>
</html>
